Dockerfile

```diff
@@ -20,6 +20,6 @@ RUN chown -R app /home/app
 USER app
 
 WORKDIR /home/app/code
-CMD node index.js | bunyan
+CMD node index.js
```

Comment:
It's best to expose raw logs, we can run `bunyan` with custom filters on raw logs, we can't anymore once they're formatted for display.